astropy/modeling/tests/test_physical_models.py

```diff
@@ -57,9 +60,12 @@ def test_blackbody_return_units():
 
 
 @pytest.mark.skipif("not HAS_SCIPY")
-def test_blackbody_fit():
-
-    fitter = LevMarLSQFitter()
+@pytest.mark.parametrize("fitter", fitters)
+def test_blackbody_fit(fitter):
+    if fitter == TRFLSQFitter:
+        pytest.xfail("TRFLSQFitter seems to be broken for this test.")
+    else:
+        fitter = fitter()
```

Comment:
@krachyon, again this test fails with the `TRFLSQFitter`, but I have not investigated why it fails yet.